otx/cli/utils/tests.py

```diff
@@ -494,32 +493,59 @@ def otx_explain_testing(template, root, otx_dir, args):
     else:
         train_type = "default"
 
-    for test_algorithm in test_algorithms:
-        save_dir = f"explain_{template.model_template_id}/{test_algorithm}/{train_type}/"
-        output_dir = os.path.join(template_work_dir, save_dir)
-        compare_dir = os.path.join(f"{otx_dir}/data/explain_samples/", save_dir)
-        command_line = [
-            "otx",
-            "explain",
-            template.model_template_path,
-            "--load-weights",
-            f"{template_work_dir}/trained_{template.model_template_id}/weights.pth",
-            "--explain-data-root",
-            os.path.join(otx_dir, args["--input"]),
-            "--save-explanation-to",
-            output_dir,
-            "--explain-algorithm",
-            test_algorithm,
-        ]
-        assert run(command_line).returncode == 0
-        for fname in os.listdir(output_dir):
-            if fname.startswith(check_files) and "saliency" in fname:
-                output_image = cv2.imread(os.path.join(output_dir, fname))
-                h, w, _ = output_image.shape
-                compare_image = cv2.imread(os.path.join(compare_dir, fname), 0)
-                compare_image = get_actmap(compare_image, (w, h))
-                diff = np.sum((compare_image - output_image) ** 2) == 0
-                assert diff == 0, f"saliency map output is not same as the sample one, with {diff}!"
+    save_dir = f"explain_{template.model_template_id}/{test_algorithm}/{train_type}/"
+    output_dir = os.path.join(template_work_dir, save_dir)
+    command_line = [
+        "otx",
+        "explain",
+        template.model_template_path,
+        "--load-weights",
+        f"{template_work_dir}/trained_{template.model_template_id}/weights.pth",
+        "--explain-data-root",
+        os.path.join(otx_dir, args["--input"]),
+        "--save-explanation-to",
+        output_dir,
+        "--explain-algorithm",
+        test_algorithm,
+    ]
+    assert run(command_line).returncode == 0
+    assert os.path.exists(output_dir)
+    assert len(os.listdir(output_dir)) > 0
+
+
+def otx_explain_openvino_testing(template, root, otx_dir, args):
+    template_work_dir = get_template_dir(template, root)
+    if "RCNN" in template.model_template_id:
+        test_algorithm = "ActivationMap"
+    else:
+        test_algorithm = "ClassWiseSaliencyMap"
+
+    train_ann_file = args.get("--train-ann-file", "")
+    if "hierarchical" in train_ann_file:
+        train_type = "hierarchical"
+    elif "multilabel" in train_ann_file:
+        train_type = "multilabel"
+    else:
+        train_type = "default"
+
+    save_dir = f"explain_{template.model_template_id}/{test_algorithm}/{train_type}/"
+    output_dir = os.path.join(template_work_dir, save_dir)
+    command_line = [
+        "otx",
+        "explain",
+        template.model_template_path,
+        "--load-weights",
+        f"{template_work_dir}/exported_nncf_{template.model_template_id}/openvino.xml",
```

Comment:
Why changed to exported NNCF IR?

Reply:
Yes, nncf is not right. Sorry for the confusion.